Gifski/util.swift

```diff
@@ -69,6 +69,14 @@ extension NSColor {
 	}
 }
 
+// MARK: - Checkerboard Colors
+extension NSColor {
+	enum Checkerboard {
+		static let first = NSColor(named: "CheckerboardFirstColor") ?? NSColor.white
+		static let second = NSColor(named: "CheckerboardSecondColor") ?? NSColor.lightGray
+	}
+}
```

Comment:
Put this in https://github.com/sindresorhus/Gifski/blob/master/Gifski/Constants.swift and you can just force-unwrap the colors.